### DISCUSSION TITLE
LIghter OPLL emulation core (emu2413)

### DISCUSSION THREAD

**Discussion by freq-mod** [LTVA1/furnace#158] [Ideas]
One extreme of the accuracy spectrum is 2608-LLE, but some systems furnace can, at least run in theory wont even handle Nuked-OPLL. Therefore, emu2413 support would be beneficial: https://github.com/digital-sound-antiques/emu2413

It's good enough, and rather fast, so can work

**Comment by LTVA1** [comment]
Hm, so add emu2413 for OPLL? And an option to switch between it or Nuked OPLL? Well, if emu2413 is faster but still good it could be a default core

**Reply by freq-mod** [reply]
It's irelevat now - tildearrow will add it himself within this week, together with other potato PC improvements
